### DISCUSSION TITLE
When I Install discord rpc in command prompt, it gets stuck on "Updating discord_rpc version git ..."

### DISCUSSION THREAD

**Discussion by Bukaboi** [KadeArchive/Kade-Engine#717] [Q&A]
![image](https://user-images.githubusercontent.com/85270024/120702536-93d9fd80-c48a-11eb-9fc8-a7ddb931d683.png)
This happens, and ive waited atleast 3 hours and nothing changes. But it says I have it installed.
![image](https://user-images.githubusercontent.com/85270024/120702742-d13e8b00-c48a-11eb-987d-f61ca611363d.png)
and when I type "lime test windows" it says i still need to install it?? Im confused. Please help

**Comment by puyoxyz** [comment]
do you have git

**Reply by dino1451** [reply]
i have git installed also and it happens to me how do i fix that?

**Comment by ghost** [comment]
Hi - Please [download git-scm](https://git-scm.com/) and try again.

**Comment by xarii** [comment]
try to remove discord_rpc and download it again
